client/my-sites/email/email-management/home/email-plan-mailboxes-list.jsx

```diff
@@ -26,15 +35,67 @@ const MailboxListHeader = ( { children } ) => {
 	);
 };
 
-const MailboxListItem = ( { children, isPlaceholder, hasNoEmails } ) => {
+const MailboxListItem = ( { children, isError = false, isPlaceholder, hasNoEmails } ) => {
 	const className = classNames( 'email-plan-mailboxes-list__mailbox-list-item', {
 		'is-placeholder': isPlaceholder,
 		'no-emails': hasNoEmails,
 	} );
-	return <CompactCard className={ className }>{ children }</CompactCard>;
+	return (
+		<CompactCard className={ className } highlight={ isError ? 'error' : null }>
+			{ children }
+		</CompactCard>
+	);
+};
+
+const MailboxListItemSecondaryDetails = ( { children, className } ) => {
+	const fullClassName = classNames( 'email-plan-mailboxes-list__mailbox-secondary', className );
+	return <div className={ fullClassName }>{ children }</div>;
+};
+
+const MailboxListItemWarning = ( { warningText } ) => {
+	return (
+		<div className="email-plan-mailboxes-list__mailbox-list-item-warning">
+			<Gridicon icon="info-outline" size={ 18 } />
+			<span>{ warningText }</span>
+		</div>
+	);
 };
 
-function EmailPlanMailboxesList( { emails, isLoadingEmails } ) {
+const resendEmailForwardVerification = ( mailbox, dispatch ) => {
+	const destination = getEmailForwardAddress( mailbox );
+	dispatch(
+		recordTracksEvent(
+			'calypso_email_management_email_forwarding_resend_verification_email_click',
+			{
+				destination,
+				domain_name: mailbox.domain,
+				mailbox: mailbox.mailbox,
+			}
+		)
+	);
+	dispatch( resendVerificationEmail( mailbox.domain, mailbox.mailbox, destination ) );
+};
+
+const getActionsForMailbox = ( mailbox, translate, dispatch ) => {
+	if ( isEmailForward( mailbox ) && ! isEmailForwardVerified( mailbox ) ) {
+		return {
+			action: (
+				<Button compact onClick={ () => resendEmailForwardVerification( mailbox, dispatch ) }>
+					{ translate( 'Resend verification' ) }
+				</Button>
+			),
+			warning: <MailboxListItemWarning warningText={ translate( 'Verification required' ) } />,
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Verification required for domains' )` *ES Score: 7*